armi/materials/material.py

```diff
@@ -582,13 +582,14 @@ def isBeyondIncubationDose(self, totalDPA):
 
         """
 
-        if not self.modelConst["Rincu"]:
+        if not "Rincu" in self.modelConst:
             msg = "Material missing incubation dose"
             runLog.warning(msg, single=True, label="Missing incubation dose")
-        elif totalDPA > self.modelConst["Rincu"]:
-            return True
         else:
-            return False
+            if totalDPA > self.modelConst["Rincu"]:
+                return True
+            else:
+                return False
```

Comment:
Simply return the result of the comparison: `return totalDPA > self.modelConst["Rincu"]`

Reply:
👍